infrastructure/pcluster-manager-cognito.yaml

```diff
@@ -202,62 +96,47 @@ Resources:
       AdminCreateUserConfig:
         AllowAdminCreateUserOnly: true
         InviteMessageTemplate:
-          EmailSubject: "[PclusterManager] Welcome to Pcluster Manager, please verify your account."
-          EmailMessage: "Thanks for installing PclusterManager on your AWS account. The following user has been created: {username}<br /><br />Please use this temporary password to login to your account: {####}"
+          EmailSubject: "[PClusterManager] Welcome to Pcluster Manager, please verify your account."
+          EmailMessage: "Thanks for installing PClusterManager on your AWS account. The following user has been created: {username}<br /><br />Please use this temporary password to login to your account: {####}"
       UsernameAttributes:
         - 'email'
       VerificationMessageTemplate:
         DefaultEmailOption: CONFIRM_WITH_CODE
-        EmailMessage: "Thanks for signing up to PclusterManager. Please use the following code to verify your account: {####}"
-        EmailSubject: "[PclusterManager] Please verify your account"
-      EmailVerificationSubject: "[PclusterManager] Please verify your account."
-      EmailVerificationMessage: "Thanks for installing PclusterManager on your AWS account. Please click the link below to verify your email address. {####}"
+        EmailMessage: "Thanks for signing up to PClusterManager. Please use the following code to verify your account: {####}"
+        EmailSubject: "[PClusterManager] Please verify your account"
+      EmailVerificationSubject: "[PClusterManager] Please verify your account."
```

Comment:
todo: as stated in the [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpool.html#cfn-cognito-userpool-emailverificationmessage), both `EmailVerificationSubject` and `EmailVerificationMessage` are no longer used and can be removed